docs/basic-features/font-optimization.md

```diff
@@ -118,9 +155,8 @@ View the [Font API Reference](/docs/api-reference/next/font.md#nextfontlocal) fo
 
 When a font function is called on a page of your site, it is not globally available and preloaded on all routes. Rather, the font is only preloaded on the related route/s based on the type of file where it is used:
 
-- if it's a [unique page](https://beta.nextjs.org/docs/routing/pages-and-layouts#pages), it is preloaded on the unique route for that page
```

Comment:
These shouldn't change

Reply:
@hanneslund I understood from this - https://github.com/vercel/next.js/pull/41788#discussion_r1006173082
That stable docs should only have `/pages` examples and beta.nextjs.org should have `/app` examples
I will double-check with Delba